src/main/java/seedu/address/MainApp.java

```diff
@@ -53,8 +50,8 @@ public void init() throws Exception {
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
         //TODO: Update this with the 4 different EntityLists
-        AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        // AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
```

Comment:
I think you should be able to immediately create an AlfredStorage object here, no need to make 4 different entitylists storage

Reply:
Ah yes this is old code. 😢